src/openforms/forms/tasks.py

```diff
@@ -160,7 +161,7 @@ def activate_forms():
                     extra={"pk": form.pk},
                 )
             else:
-                transaction.on_commit(lambda: logevent.form_activated(form))
+                transaction.on_commit(partial(logevent.form_activated, form))
```

Comment:
Is there a difference between these two?Isn't the result the same?

Reply:
https://docs.astral.sh/ruff/rules/function-uses-loop-variable/ ;)